src/components/base/JTextArea/JTextArea.js

```diff
@@ -1,133 +1,182 @@
-// @flow
+// @flow strict
 
+import classNames from 'classnames'
 import React, { PureComponent } from 'react'
+
 import {
   noop,
   omit,
   kebabCase,
 } from 'lodash-es'
-import classNames from 'classnames'
+
+import { getErrorMessage } from 'utils/form'
+import { JFieldMessage } from 'components/base'
 
 import jTextAreaStyle from './jTextArea.m.scss'
 
 type Theme = 'white'
 
-type Props =
-  StyleComponent<Theme>
-  & {
-    onChange?: () => mixed,
-    onBlur?: () => mixed,
-    onFocus?: () => mixed,
-    label?: string,
-    id?: string,
-    value?: any, // In common case it will be string, but in React value of input has type `any`
-    rows?: number,
-    error?: boolean,
-    disabled?: boolean,
-  }
+type Props = StyleComponent<Theme> & {
+  +onChange: (?SyntheticEvent<HTMLTextAreaElement>) => void,
```

Comment:
if we suggesting that this property of record is not required, I would prefer use `onChange?: ...`, in case if we suggesting that property is required, but handler is not, I would prefer use `defaultProps` with value `noop`